packages/ui/cypress/support/page-objects/multisigPage.ts

```diff
@@ -1,5 +1,5 @@
 export const multisigPage = {
-  accountHeader: () => cy.get('[data-cy=header-account]'),
+  accountHeader: (timeout = 4000) => cy.get('[data-cy=header-account]', { timeout }),
```

Comment:
I think this one can be removed now, it is declaring the default anyway

Reply:
nah, it's a JS thing, that let you define a param, and if it's undefined, then it'll be 4000. I actually use it in the creation to wait for 10s `multisigPage.accountHeader(10000)`. Let me check if things pass without.